app/views/layouts/application.html.haml

```diff
@@ -42,12 +45,24 @@
                 %li= link_to 'Francais', params.merge({ locale: :fr })
                 %li= link_to 'Italiano', params.merge({ locale: :it })
                 %li= link_to 'English', params.merge({ locale: :en })
-
+    -# Content form different pages start here
```

Comment:
Comments in HAML can be made with /
Hence: 
`/ Content form different pages start here`